### ISSUE TITLE
Activating a profile triggers a warning

### ISSUE THREAD

**Issue by dgageot** [GoogleContainerTools/skaffold#1688]
```
$ skaffold build -p anyprofile
WARN[0000] unknown field type in profile overlay: ptr. falling back to original config values
```

This is because the code that merges profiles into main configuration doesn't handle some recent fields that are pointers, such as `ExecutionEnvironment *ExecutionEnvironment`.

**Comment by demisx** [comment]
I am seeing this warning in the latest skaffold v0.36 on Mac Os Mojave 10.14.6 running any skaffold command that reference a profile:

```sh
WARN[0000] unknown field type in profile overlay: int. falling back to original config values
```

**Comment by kurczynski** [comment]
I'm also seeing this in version `0.36` on Fedora 30:
```
WARN[0000] unknown field type in profile overlay: int. falling back to original config values
```

**Comment by bjesus** [comment]
Also seeing this on ArchLinux with Skaffold 0.36. Perhaps this issue should be reopened? @dgageot

**Comment by dgageot** [comment]
@demisx @kurczynski @bjesus Could you provide a sample that shows the issue? Or maybe it happens with one of our samples?

**Comment by dgageot** [comment]
This has been fixed in 83b0f45bf3db659f942d1c0d40cd93bfdd44d3ad, just after the 0.36 release freeze.
